### ISSUE TITLE
Support shortcut Ctrl+/ to quickly comment/uncomment selected lines

### ISSUE THREAD

**Issue by yqs112358** [thomiceli/opengist#379]
The text-only opengist is ideal for hosting configuration files. So, it would be very helpful to create & edit configurations if the shortcut Ctrl+/ could be used to quickly comment/uncomment selected lines, just as what works on common IDEs like VSCode.

**Comment by thomiceli** [comment]
I agree the CodeMirror editor in Opengist could be more user friendly, maybe we could add other shortcuts ?

**Comment by GGORG0** [comment]
> I agree the CodeMirror editor in Opengist could be more user friendly, maybe we could add other shortcuts ?

Is it just me or does the editor not have any syntax highlighting or anything special? It's just a simple input for me...

**Comment by thomiceli** [comment]
> > I agree the CodeMirror editor in Opengist could be more user friendly, maybe we could add other shortcuts ?
> 
> Is it just me or does the editor not have any syntax highlighting or anything special? It's just a simple input for me... I thought CodeMirror had at least syntax highlighting, right?

You're right this is not something i spent a lot of time on until now, it always has room for improvement :)
